### PR TITLE
Added sb-admin-2 bootstrap theme for admin

### PR DESCRIPTION
**Pull request by skarm61**

### CODE REVIEW

**Comment by kont-noor** [comment]
Specs fail

```
20 examples, 5 failures
```

```
  5) posts show post display post by slug
     Failure/Error: visit("/yablog/category/slug")
     ActionView::Template::Error:
       couldn't find file 'metisMenu'
```

File: app/views/layouts/yablog/admin.html.haml [hunk 1]
```diff
@@ -2,23 +2,29 @@
 %html
   %head
     %title WebLog
-    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
-    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = stylesheet_link_tag    'yablog/application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'yablog/application', 'data-turbolinks-track' => true
```
Comment (kont-noor) [inline app/views/layouts/yablog/admin.html.haml]: What about to use separate css and js here?

File: app/views/yablog/admin/shared/_sidebar.html.haml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.sidebar-nav.navbar-collapse
+  %ul#side-menu.nav
+    / %li.sidebar-search
```
Comment (DevMike) [inline app/views/yablog/admin/shared/_sidebar.html.haml]: it would make sense to leave some comments why exactly it was commented out to not bring garbage

**Comment by kont-noor** [comment]
@skarm61 

fixed it by the following diff in `Gemfile`:

```diff
 source "https://rubygems.org"
+source 'https://rails-assets.org'
```

**Comment by kont-noor** [comment]
Also add normal show view

**Comment by skarm61** [comment]
addressed in c8fa031

**Comment by kont-noor** [comment]
@skarm61 check the frontend now.

**Pull request merged**